### ISSUE TITLE
Tutorials/Examples under a different namespace

### ISSUE THREAD

**Issue by MasterSkepticista** [intel/openfl#450]
To ensure velocity of changes/features in mainline, updating examples in parallel to comply with the release would consume time.

Suggestion is to transition tutorials/examples under `openfl-examples` repository, which has its release tags tied to the last tested working version of OpenFL even if the `develop` branch treads ahead to address priority features that may break some examples.

**Comment by MasterSkepticista** [comment]
@psfoley @mansishr @shreyasnc1 @igor-davidyuk @acharyasunil

**Comment by mkurisoo** [comment]
I agree, tags on the tutorials of the last tested working version of OpenFL seems like a good idea.

**Comment by rahulga1** [comment]
I think, we can taken up the cleanup activity and this will continue happening over the time. So, closing for now.
